### PR TITLE
[Rule Tuning] Interactive Terminal Spawned via Python

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Brings https://github.com/elastic/detection-rules/pull/1649 back to life now that we support type changes

### CODE REVIEW

File: rules/linux/execution_python_tty_shell.toml [hunk 1]
```diff
@@ -24,14 +24,25 @@ tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "Elastic En
 timeline_id = "e70679c2-6cde-4510-9764-4823df18f7db"
 timeline_title = "Comprehensive Process Timeline"
 timestamp_override = "event.ingested"
```
Comment (Aegrah) [inline rules/linux/execution_python_tty_shell.toml:26]:
```suggestion
```

File: rules/linux/execution_python_tty_shell.toml [hunk 2]
```diff
@@ -24,14 +24,25 @@ tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "Elastic En
...
+      "/bin/sh",
+      "/bin/dash",
+      "/bin/csh",
+      "/bin/zsh",
+      "/bin/ksh",
+      "/bin/tcsh",
+      "/bin/fish",
+      "/bin/ash",
+      "/bin/bsh"
+   )
```
Comment (Aegrah) [inline rules/linux/execution_python_tty_shell.toml:44]:
```suggestion
   process.executable : "/bin/*sh"
```

**Review by Aegrah — APPROVED** [review]
Unit testing fix and a potential change of readibility / less easy to bypass (as most shell equivalents in Linux end with sh). Other than that, LGTM

**Review by imays11 — APPROVED** [review]

**Comment by okbud12** [comment]
@w0rk3r the updated rule seems to be creating a lot of false positives. It is being triggered by services that perform health checks and configuration checks. 

For example, OMS Agent in linux runs a python script which collects telemetry. This further spawns a bash shell that displays the process metrics. 
Similarly, there are other more services related to Hadoop cluster monitoring that are creating same. 

Not sure if this would be the ideal optimization but there could be a NOT process.command_line: (#All the commands to ignore) added in the query as well.

```
sequence with maxspan=1m
  [process where host.os.type == "linux" and event.type == "start" and process.name : "python*"] by process.entity_id
  [process where host.os.type == "linux" and event.type == "start" and 
   process.executable : "/bin/*sh"
   and process.command_line != " "
  ] by `process.parent.entity_id`
```

**Comment by w0rk3r** [comment]
@okbud12, thanks for reporting, I'll take a look at it and push a tuning

**Pull request merged**